<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Linux Process Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="freezeCheckBox">
         <property name="text">
          <string>freeze</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="memory">
         <property name="toolTip">
          <string>Memory usage</string>
         </property>
         <property name="styleSheet">
          <string notr="true"> QProgressBar {
     border: 2px solid grey;
     border-radius: 3px;
 }

 QProgressBar::chunk {
     background-color:rgb(32, 140, 2)
 }

 QProgressBar {
     border: 2px solid grey;
     border-radius: 3px;
     text-align: center;
 }

 </string>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="swap">
         <property name="toolTip">
          <string>Swap usage</string>
         </property>
         <property name="styleSheet">
          <string notr="true"> QProgressBar {
     border: 2px solid grey;
     border-radius: 3px;
 }

 QProgressBar::chunk {
     background-color:rgb(185, 0, 3)
 }

 QProgressBar {
     border: 2px solid grey;
     border-radius: 3px;
     text-align: center;
 }

 </string>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PlotWidget" name="plotOverallCpuHist"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTreeWidget" name="processTreeWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="headerDefaultSectionSize">
       <number>50</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose_all_and_exit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionShow_process_from_all_users"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSystem_information"/>
    <addaction name="actionNetwork_Information"/>
   </widget>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionSet_affinityPriority"/>
    <addaction name="separator"/>
    <addaction name="actionKill_process"/>
    <addaction name="actionKill_process_tree"/>
    <addaction name="actionSuspend_process"/>
    <addaction name="separator"/>
    <addaction name="actionProperties"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionSaveSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionColor_legend"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionLog"/>
    <addaction name="actionClear_Messages"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuView"/>
   <addaction name="menuProcess"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSet_affinityPriority">
   <property name="text">
    <string>Set affinity / priority</string>
   </property>
  </action>
  <action name="actionSet_priority">
   <property name="text">
    <string>Set priority</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>----</string>
   </property>
  </action>
  <action name="actionKill_process">
   <property name="text">
    <string>Kill process</string>
   </property>
  </action>
  <action name="actionKill_process_tree">
   <property name="text">
    <string>Kill process tree</string>
   </property>
  </action>
  <action name="actionSuspend_process">
   <property name="text">
    <string>Suspend process</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties</string>
   </property>
  </action>
  <action name="actionShow_process_from_all_users">
   <property name="text">
    <string>Show process from all users</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action9">
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="action7">
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action8">
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action10">
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action12">
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="action14">
   <property name="text">
    <string>14</string>
   </property>
  </action>
  <action name="actionSaveSettings">
   <property name="text">
    <string>Save current settings</string>
   </property>
  </action>
  <action name="action10_2">
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action100">
   <property name="text">
    <string>100</string>
   </property>
  </action>
  <action name="action1000">
   <property name="text">
    <string>1000</string>
   </property>
  </action>
  <action name="action10000">
   <property name="text">
    <string>10000</string>
   </property>
  </action>
  <action name="action100000">
   <property name="text">
    <string>100000</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionSystem_information">
   <property name="text">
    <string>System Information...</string>
   </property>
  </action>
  <action name="actionNetwork_Information">
   <property name="text">
    <string>Network Information...</string>
   </property>
  </action>
  <action name="actionClose_this_window">
   <property name="text">
    <string>Close this window</string>
   </property>
  </action>
  <action name="actionClose_all_and_exit">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionColor_legend">
   <property name="text">
    <string>Color legend</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>---</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionClear_Messages">
   <property name="text">
    <string>Clear 'do not show again' messages</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
